src/EPR.RegulatorService.Frontend.Web/Controllers/Registrations/RegistrationsController.cs

```diff
@@ -267,6 +271,111 @@ public async Task<IActionResult> AcceptRegistrationSubmission(AcceptRegistration
 
         }
 
+        [HttpGet]
+        [Route(PagePath.OrganisationDetailsFileDownload)]
+        public IActionResult OrganisationDetailsFileDownload(bool downloadFailed = false, bool hasVirus = false, bool hasFile = false)
```

Comment:
@scdefra Hi Simon, thank you for your review, I tried to follow that way of calling the view but in MVC the view needs to have a method linked to it. when we delete  OrganisationDetailsFileDownload method, it breaks the flow and we don't see OrganisationDetailsFileDownload.cshtml because there is no corresponding method. 

However, other than that, I have refactored the other parts of the code and removed few unnecessary lines from views and controller as you and me discussed over the call on Friday. Also added the Mocked version of downloadfile method,  and now we have increased the code coverage as well.

Reply:
new PR has been created from this Branch https://github.com/DEFRA/epr-regulator-service/pull/19/files

Reply:
Resolved in PR